packages/graphql-codegen-cli/src/config.ts

```diff
@@ -77,6 +74,8 @@ function customLoader(ext: 'json' | 'yaml' | 'js' | 'ts') {
     }
 
     if (ext === 'ts') {
+      // #8437: conflict with `graphql-config` also using TypeScriptLoader(), causing a double `ts-node` register.
```

Comment:
@Diizzayy, doing so will register ts-node multiple times 😣

Reply:
@charlypoly Thank you for taking the time to look at this, I must've overlooked something when I tested for the double ts-node registration.

I'm hoping desperately that we'll be able to come up with a workaround

Reply:
@charlypoly I must admit that I'm a bit confused here, In my tests thus far, my proposed change doesn't reintroduce the issue raised in #8437 and I can't seem to detect double `ts-node` registrations. Can you have another look into this or possibly direct me as to how to properly check for the ts-node issue